docs/paper.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'scores: A Python package for verifying accuracy using xarray and pandas'
+tags:
+  - Python
+  - geoscience
+  - verification
+  - science
+  - earth system science
+  - statistics
+authors:
+  - name: Tennessee Leeuwenburg
+    orcid: 0009-0008-2024-1967
+    affiliation: 1 
+    corresponding: true # (This is how to denote the corresponding author)    
+  - name: Nicholas Loveday
+    affiliation: 1
+affiliations:
+ - name: Bureau of Meteorology, Australia
+   index: 1
+date: 6 September 2023
+bibliography: paper.bib 
+
+---
+
+# Summary
+
+`scores` is a Python package containing mathematical functions for the verification, evaluation, and optimisation of model outputs and predictions. It primarily supports the geoscience and earth system science communities. `scores` is focused on supporting xarray datatypes for earth system data. It has wide potential application in machine learning, and domains other than meteorology, geoscience and weather. It also aims to be compatible with xarray, pandas, geopandas and work with NetCDF4, Zarr, hdf5 and GRIB data sources among others. Scores is designed to utilise Dask for scaling and performance.
```

Comment:
```suggestion
`scores` is a Python package containing mathematical functions for the verification, evaluation, and optimisation of model outputs and predictions. It primarily supports the geoscience and earth system science communities. `scores` is focused on supporting xarray datatypes for earth system data. It has wide potential application in machine learning, and domains other than meteorology, geoscience and weather. It also aims to be compatible with pandas, geopandas and work with NetCDF4, Zarr, hdf5 and GRIB data sources among others. Scores is designed to utilise Dask for scaling and performance.
```
Delete the word xarray since you already say that it supports xarray

Reply:
done